doc-content/kogito-docs/src/main/asciidoc/dmn/con_dmn-documentation.adoc

```diff
@@ -0,0 +1,7 @@
+[id='con_dmn-documentation_{context}']
+= DMN model documentation in the {PRODUCT} DMN designer
+
+In the {PRODUCT} DMN designer, you can use the *Documentation* tab to generate a report of your DMN model that you can print or download as an HTML file for offline use. The DMN model report contains all decision requirements diagrams (DRDs), data types, and boxed expressions in your DMN model. You can use this report to share your DMN model details or as part of your internal reporting workflow.
```

Comment:
`kogito` does not support printing or downloading the documentation.

See https://issues.redhat.com/browse/KOGITO-1156

Reply:
@manstis, oops! I saw that change in the latest UI and updated the screenshot but forgot to change the text. :) Good catch. Fixed. Thank you!